webui/react/README.md

```diff
@@ -46,6 +46,13 @@ This is set to `/det` by default for typical workflows in this project.  [More i
 - `DET_NODE_ENV`: set this to `development` to skip some build optimizations when developing and building
 locally to lower build time.
 
+## Developing Against a Remote Cluster
+
+If the remote cluster has `enable_cors` set or allows CORS requests set `SERVER_ADDRESS` to point
```

Comment:
took me a while to understand this paragraph (maybe my bad english):
- `has 'enable_cors' set` to any value?
- maybe put a comma `allows CORS requests, set ...`

Reply:
both good points, thanks